crates/ruff_python_formatter/generate.py

```diff
@@ -38,9 +38,10 @@ def rustfmt(code: str) -> str:
     # implementation.
```

Comment:
You might want to update this comment or maybe generalize it to state that formatting for these nodes are handled by another node.